src/pretix/plugins/banktransfer/tasks.py

```diff
@@ -162,6 +162,21 @@ def _handle_transaction(trans: BankTransaction, matches: tuple, event: Event = N
     else:
         trans.order = orders[0]
 
+    if len(orders) > 1:
+        # Multi-match! Can we split this automatically?
+        order_pending_sum = sum(o.pending_sum for o in orders)
+        if order_pending_sum != trans.amount:
+            # we can't :( this needs to be dealt with by a human
+            trans.state = BankTransaction.STATE_NOMATCH
+            trans.message = gettext_noop('Automatic split to multiple orders not possible.')
```

Comment:
Wollen wir ggf. noch in die Message schreiben, welche Orders erkannt wurden? In `trans.order` steht ja nur die erste drin, was etwas verwirrend sein könnte

```suggestion
            trans.message = gettext_noop('Automatic split to multiple orders not possible.') + \
                ' (' +', '.join(order.full_code for order in orders) + ')'
```

Reply:
Nette Idee, aber bricht die Lokalisierung. Wir jagen das hier ja nur durch gettext_noop, in der Datenbank steht also immer englisch, und dann im Template später nochmal {% trans trans.message %}. Ist eh hacky, aber dann würde es gar nicht mehr tun. Ist daher den Zusatzaufwand denke ich nicht wert.